test/copyPaste.test.tsx

```diff
@@ -45,13 +45,19 @@ const summaryRows: readonly Row[] = [
 const copyCellClassName = 'rdg-cell-copied';
 const onPasteSpy = jest.fn();
 const onCopySpy = jest.fn();
+const externalPasteSpy = jest.fn();
+const externalCopySpy = jest.fn();
 
 function CopyPasteTest({
   onPasteCallback = true,
-  onCopyCallback = false
+  onCopyCallback = false,
+  externalCopy = false,
+  externalPaste = false,
 }: {
   onPasteCallback?: boolean;
   onCopyCallback?: boolean;
+  externalCopy?: boolean;
+  externalPaste?: boolean;
```

Comment:
I would try to keep the "on" convention they have in the previous callers, just to keep aligned with their code style.

Code looks good and thanks for this improvement, I am looking forward to this change too.